packages/reports/app/styles/navi-reports/components/grouped-list.less

```diff
@@ -1,27 +1,115 @@
 /**
- * Copyright 2018, Yahoo Holdings Inc.
+ * Copyright 2019, Yahoo Holdings Inc.
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 .grouped-list {
-  color: @navi-gray-500;
+  color: @denali-grey-800;
   font-size: @font-size-mid-base;
 
   &__group-header {
-    text-transform: capitalize;
     margin: 2px 0;
+
+    &-content {
+      font-family: @font-family-sans-serif; // override icon font
+      text-transform: capitalize;
+    }
+
+    &::-webkit-details-marker {
+      display: none; // hide default <summary> marker in Chrome
```

Comment:
Can we check the rendering of this on different browsers? Im not sure if they have updated the browsers, but I noticed a few differences between chrome and firefox when I built this initially.

Reply:
Checked on Chrome and Firefox, any other?

Reply:
Safari?

Reply:
Looks good on Safari